cli/serve/serve.go

```diff
@@ -94,9 +95,23 @@ var v1Endpoints = map[string]func() (http.Handler, error){
 	"scaninfo": func() (http.Handler, error) {
 		return scan.NewInfoHandler(), nil
 	},
+}
 
+var staticEndpoints = map[string]func() (http.Handler, error){
 	"/": func() (http.Handler, error) {
-		return http.FileServer(rice.MustFindBox("static").HTTPBox()), nil
+		//the default rice order doesn't include working directory
```

Comment:
This wasn't an intended use case (if the cfssl source has been moved, it would really be ideal to [build the binary with any static file embedded](https://github.com/cloudflare/cfssl/blob/master/README.md#static-builds)). But, it does seem fine to add the working directory as a final fallback.